### PR TITLE
Commit Gemfile.lock to source control

### PR DESCRIPTION
**Pull request by deivid-rodriguez**
As agreed and requested in activeadmin/inherited_resources#453.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 language: ruby
 sudo: false
+before_install:
+  - gem update --system 2.7.7
```
Comment (javierjulio) [inline .travis.yml]: I haven’t seen this before. Is this installing a specific version of rubygems? If so I see why so it’s consistent for all builds. Normally though it’s the most recent. Are we sure we want this? I take it we’d have to do this in other repos too?

**Comment by deivid-rodriguez** [comment]
I added java platform to the lock file with `bundle lock --add-platform java` and it's passing now. The `gem update --system` line is optional for getting this green but I think it's good since it's brings more determinism to the build.

File: .travis.yml (hunk 1 above)
Comment (javierjulio) [reply .travis.yml]: @deivid-rodriguez perhaps its not a big deal, but I noticed the `before_install` section here does not install the latest bundler [like we do in activeadmin](https://github.com/activeadmin/activeadmin/blob/96b22bf08a879dc856aee0f7396d2a70fbc65209/.travis.yml#L29). Do you think we should have that here and be consistent or just this as is? Let me know what you think as besides that I think this looks good to go. 👍🏻

**Comment by deivid-rodriguez** [comment]
I guess it makes sense, yeah. But since the point of this PR was only to add lock files to the repo, I ended up actually removing the `rubygems` update. We install latest bundler and rubygems in a separate PR if we feel it's necessary.

**Comment by deivid-rodriguez** [comment]
Going in!

**Pull request merged**